mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml

```diff
@@ -11,6 +11,59 @@
         <default>0</default>
         <Required>Y</Required>
       </enable_redis_plugin>
+      <rejectscore type="IntegerField">
+        <Required>N</Required>
+        <MinimumValue>1</MinimumValue>
+        <MaximumValue>999</MaximumValue>
+        <Constraints>
+          <check001>
+            <ValidationMessage>This field must be bigger than the header score.</ValidationMessage>
+            <type>ComparedToFieldConstraint</type>
+            <field>headerscore</field>
+            <operator>gt</operator>
+          </check001>
+          <check002>
+            <ValidationMessage>This field must be lower than the reject score.</ValidationMessage>
+            <type>ComparedToFieldConstraint</type>
+            <field>rejectscore</field>
+            <operator>lt</operator>
+          </check002>
+          <check003>
+            <ValidationMessage>This field must be bigger than the greylist score.</ValidationMessage>
+            <type>ComparedToFieldConstraint</type>
+            <field>greylistscore</field>
+            <operator>gt</operator>
+          </check003>
+        </Constraints>
+      </subjectscore>
```

Comment:
but closes as "subjectscore" @fabianfrz @mimugmail this is pretty much broken at the moment.. "make lint" does catch this ;)

Reply:
I'll fix this, have not found it while reviewing...